multiqc/modules/nonpareil/nonpareil.py

```diff
@@ -0,0 +1,295 @@
+            },
+            "kappa": {
+                "title": "Redundancy",
+                "description": "Dataset redundancy",
+                "modify": lambda x: x * 100,
+                "max": 100,
+                "min": 0,
+                "suffix": "%",
+                "scale": "RdYlGn-rev",
+                "format": "{:,.2f}",
+            },
+            "C": {
+                "title": "Coverage",
+                "description": "Dataset coverage",
+                "modify": lambda x: x * 100,
+                "max": 100,
+                "min": 0,
+                "suffix": "%",
+                "scale": "RdYlGn",
+                "format": "{:,.2f}",
+            },
+            "consistent": {
+                "title": "Consistent Data?",
+                "description": "Is the data sufficient for accurate estimation?",
+                "modify": lambda x: "Yes" if x == 1 else "No",
+                "scale": False,
+                "hidden": True,
+            },
+            "star": {
+                "title": "Ideal coverage",
+                "description": "Objective coverage in percentage; i.e., coverage value considered near-complete.",
+                "max": 100,
+                "min": 0,
+                "suffix": "%",
+                "scale": False,
+                "hidden": True,
+            },
+            "LRstar": {
+                "title": f"{config.base_count_prefix} Sequencing effort for ideal coverage",
+                "description": f"Projected sequencing effort for nearly complete coverage ({config.base_count_desc})",
+                "modify": lambda x: x * config.base_count_multiplier,
+                "min": 0,
+                "scale": "RdYlGn-rev",
+                "format": "{:,.2f} " + config.base_count_prefix,
+                "shared_key": "base_count",
+            },
+            "modelR": {
+                "title": "Model correlation",
+                "description": "Pearson R for the estimated model",
+                "max": 1,
+                "min": 0,
+                "scale": "RdYlGn",
+                "format": "{:,.3f}",
+                "hidden": True,
+            },
+            "diversity": {
+                "title": "Diversity",
+                "description": "Dataset Nd index of sequence diversity",
+                "min": 0,
+                "scale": "RdYlGn",
+                "format": "{:,.2f}",
+            },
+        }
+
+        self.general_stats_addcols(self.data_by_sample, headers)
+
+    def nonpareil_redundancy_plot(self):
+        """Make the redundancy plot for nonpareil"""
+
+        esconfig = {
+            "dashStyle": "Dash",
+            "lineWidth": 2,
+            "color": "#000000",
+            "marker": {"enabled": False},
+            "enableMouseTracking": False,
+            "showInLegend": False,
+        }
+        # desc = " **The dashed black line shows theoretical GC content:** `{}`".format(theoretical_gc_name)
```

Comment:
Please remove, and add a description of what the line represents to the section description (no need to have it as a variable as done in the FastQC module, as this one is always shown - the FastQC one is optional).